allensdk/brain_observatory/session_api_utils.py

```diff
@@ -152,7 +152,7 @@ def clear_updated_params(self, data_params: set):
         self._updated_params -= data_params
 
 
-def sessions_are_equal(A, B, reraise=False) -> bool:
+def sessions_are_equal(A, B, reraise=False, skip_field=set()) -> bool:
```

Comment:
Be aware that initializing `skip_field` with a default has possibly unexpected behavior:
https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments

Reply:
You know... I used to know that. I'll fix it.